src/arch/freeRTOS/main.cpp

```diff
@@ -13,31 +13,38 @@
 #include "FreeRTOS.h"
 #include "task.h"
 
-#include "../forte_architecture.h"
-#include "../devlog.h"
-#include "forteinstance.h"
+#include "devlog.h"
+#include "../c_interface/forte_c.h"
 
-const static unsigned mainFORTE_TASK_PRIORITY = tskIDLE_PRIORITY + 1;
+static const unsigned scForteTaskPriority = tskIDLE_PRIORITY + 1;
+
+static const unsigned int scDesiredFortePort = 61499;
```

Comment:
Please correct me if I am wrong, but didn't we agree to move this into an unnamed namespace and not to use prefixes here?

Reply:
you're right. It was in a similar file, but not this one. I updated this one for consistency.